plugins/services/src/js/validators/PlacementsValidators.js

```diff
@@ -138,6 +140,12 @@ const PlacementsValidators = {
 
       return errors;
     }, []);
+  },
+
+  validateNoBatchError(constraintTransactions) {
```

Comment:
🎨  Let's move this one to it's own util as it seems to be a rather general utility.

Reply:
Also don't agree much here, because we will make general something that is not at this moment. I believe if/when we make the batch error standard, then that could be refactored to its own utility. Right now it only servers for Placement Constraints.